userspace/libscap/userspace_flag_helpers.h

```diff
@@ -53,16 +53,6 @@ limitations under the License.
 #define u32 uint32_t
 #define u64 uint64_t
 
-#ifndef UDIG
-#define UNDEF_UDIG
-#define UDIG
-#endif
```

Comment:
I know what this is for and removing it could break gVisor. I see that in some cases thre is some logic with `#ifdef __KERNEL__` so it might be already handled, but have you tried building gVisor support?

Reply:
yes, also the CI is running with gvisor enabled

Reply:
I moved the necessary imports into the `ppm_flag_helpers.h` file

Reply:
Thanks Andrea